packages/turf-collect/.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
+node_modules/
+npm-debug.log
+
```

Comment:
The `.gitignore` is in the root directory, no need to include this.

https://github.com/Turfjs/turf/blob/master/.gitignore

```
lib-cov
*.seed
*.log
*.csv
*.dat
*.out
*.pid
.idea/*
types.js
.DS_Store

pids
logs
results

npm-debug.log*

node_modules
```